rfc: 1-animation-slide-side

Ciflire opened the pull request:
I find the implementation wierd with sides.
Decided to use this space to get feedbacks

Ciflire committed:
rfc: 1-animation-slide-side

vaxerski commented:
@NotAShelf is this important enough for an RFC? I think this should just be an enhancement request in hl

NotAShelf commented:
It could be a topic for the RFC if it proposed an entire refactor, but it completely ignores the RFC format.

NotAShelf commented:
"hey can we do this" is not a good RFC topic.

A good RFC

1. Describes the underlying issue
2. Convinces the reader that it is in fact an issue and not a nitpick
3. Proposes changes, and possibly the implementation

Ciflire commented:
Could you let me qualify this as a draft and work on it using your feedbacks?

NotAShelf commented:
Do feel free. I'll provide a quick review tomorrow.

Ciflire committed:
rfc: detailed motivation

beside the motivation the rest wasn't modified yet but will be

Ciflire committed:
rfc: added refs, rewrote it

Some sections are still empty, i don't know what to put it there by
reading the template. But I will fill these up if remarks/questions are
asked.

Ciflire commented:
> is this important enough for an RFC

I don't know if it is, but it's already implemented and it seems like i have a diverging point of view from the implementation, I thought the best way to settle this was an RFC
